script.js

```diff
@@ -299,3 +310,45 @@ workMobile.forEach((work) => {
   });
   mobileWork.appendChild(item);
 });
+
+// form validatio
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224448188-33e684d9-931a-46a5-8233-dfe27b19bbc0.png)

**[Optional]**

- [ ] It would be nice if you make all your commit messages meaningful and capitalize to make your pull request descriptive and professional 😊